k4EDM4hep2LcioConv/include/k4EDM4hep2LcioConv/k4Lcio2EDM4hepConv.ipp

```diff
@@ -174,6 +176,29 @@ std::vector<CollNamePair> convertReconstructedParticles(const std::string& name,
   return results;
 }
 
+// Go from chi^2 and probability (1 - CDF(chi^2, ndf))
+// to ndf by a binary search
+int find_ndf(double chi2, double prob) {
```

Comment:
Since this is not a template, we could put this into the header (and potentially into a nested namespace) and move the implementation to the `cc` file.

Reply:
Done, I think a namespace for a single small function is a bit overkill, no?